src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -29,18 +31,21 @@ internal class BlobsExtensionConfigProvider : IExtensionConfigProvider,
         private IContextGetter<IBlobWrittenWatcher> _blobWrittenWatcherGetter;
         private readonly INameResolver _nameResolver;
         private IConverterManager _converterManager;
+        private readonly ILogger _logger;
 
         public BlobsExtensionConfigProvider(StorageAccountProvider accountProvider,
             BlobTriggerAttributeBindingProvider triggerBinder,
             IContextGetter<IBlobWrittenWatcher> contextAccessor,
             INameResolver nameResolver,
-            IConverterManager converterManager)
+            IConverterManager converterManager,
+            ILoggerFactory loggerFactory)
         {
             _accountProvider = accountProvider;
             _triggerBinder = triggerBinder;
             _blobWrittenWatcherGetter = contextAccessor;
             _nameResolver = nameResolver;
             _converterManager = converterManager;
+            _logger = loggerFactory?.CreateLogger(LogCategories.Bindings);
```

Comment:
Create a new category : `Host.Binding.AccessStats` ? this will make it easier for your queries to filter by `Source` column in Kusto